src/main/java/uk/gov/hmcts/reform/blobrouter/services/BlobVerifier.java

```diff
@@ -47,8 +48,10 @@ public boolean verifyZip(String blobName, byte[] rawBlob) {
             logger.info("Invalid zip archive. Blob name: {}", blobName, ex);
             return false;
         } catch (IOException ex) {
-            logger.info("Error occurred when verifying file. Blob name: {}", blobName, ex);
-            return false;
+            throw new BlobProcessingException(
```

Comment:
Should this happen for all other exceptions, i.e. should we change the exception type in this `catch` to `Exception`?

Reply:
Plus, if we make it `Exception`, we can test that `BlobVerifier` throws `BlobProcessingExceptions` when it fails to verify the file.